workers/loc.api/sync/dao/db-migrations/sqlite-migrations/migration.v19.js

```diff
@@ -0,0 +1,353 @@
+      'DROP INDEX statusMessages_timestamp_key',
+      'DROP INDEX candles_mts__symbol',
+
+      `CREATE INDEX ledgers_user_id_wallet_currency_mts
+        ON ledgers(user_id, wallet, currency, mts)`,
+      `CREATE INDEX ledgers_user_id_wallet_mts
+        ON ledgers(user_id, wallet, mts)`,
+      `CREATE INDEX ledgers_user_id_currency_mts
+        ON ledgers(user_id, currency, mts)`,
+      `CREATE INDEX ledgers_user_id__isMarginFundingPayment_mts
+        ON ledgers(user_id, _isMarginFundingPayment, mts)`,
+      `CREATE INDEX ledgers_user_id__isAffiliateRebate_mts
+        ON ledgers(user_id, _isAffiliateRebate, mts)`,
+      `CREATE INDEX ledgers_user_id__isStakingPayments_mts
+        ON ledgers(user_id, _isStakingPayments, mts)`,
+      `CREATE INDEX ledgers_user_id__category_mts
+        ON ledgers(user_id, _category, mts)`,
+      `CREATE INDEX ledgers_user_id_mts
+        ON ledgers(user_id, mts)`,
+      `CREATE INDEX ledgers_currency_mts
+        ON ledgers(currency, mts)`,
+      `CREATE INDEX ledgers_user_id_subUserId_mts
+        ON ledgers(user_id, subUserId, mts)
+        WHERE subUserId IS NOT NULL`,
+      `CREATE INDEX ledgers_subUserId_mts__id
+        ON ledgers(subUserId, mts, _id)
+        WHERE subUserId IS NOT NULL`,
+
+      `CREATE INDEX trades_user_id_symbol_mtsCreate
+        ON trades(user_id, symbol, mtsCreate)`,
+      `CREATE INDEX trades_user_id_orderID_mtsCreate
+        ON trades(user_id, orderID, mtsCreate)`,
+      `CREATE INDEX trades_user_id_execAmount_mtsCreate
+        ON trades(user_id, execAmount, mtsCreate)`,
+      `CREATE INDEX trades_user_id_execPrice_mtsCreate
+        ON trades(user_id, execPrice, mtsCreate)`,
+      `CREATE INDEX trades_user_id_mtsCreate
+        ON trades(user_id, mtsCreate)`,
+      `CREATE INDEX trades_user_id_subUserId_mtsCreate
+        ON trades(user_id, subUserId, mtsCreate)
+        WHERE subUserId IS NOT NULL`,
+      `CREATE INDEX trades_subUserId_orderID
+        ON trades(subUserId, orderID)
+        WHERE subUserId IS NOT NULL`,
+
+      `CREATE INDEX fundingTrades_user_id_symbol_mtsCreate
+        ON fundingTrades(user_id, symbol, mtsCreate)`,
+      `CREATE INDEX fundingTrades_user_id_mtsCreate
+        ON fundingTrades(user_id, mtsCreate)`,
+      `CREATE INDEX fundingTrades_user_id_subUserId_mtsCreate
+        ON fundingTrades(user_id, subUserId, mtsCreate)
+        WHERE subUserId IS NOT NULL`,
+
+      `CREATE INDEX publicTrades__symbol_mts
+        ON publicTrades(_symbol, mts)`,
+      `CREATE INDEX publicTrades_amount_mts
+        ON publicTrades(amount, mts)`,
+      `CREATE INDEX publicTrades_mts
+        ON publicTrades(mts)`,
+
+      `CREATE INDEX orders_user_id_symbol_mtsUpdate
+        ON orders(user_id, symbol, mtsUpdate)`,
+      `CREATE INDEX orders_user_id_type_mtsUpdate
+        ON orders(user_id, type, mtsUpdate)`,
+      `CREATE INDEX orders_user_id_amount_mtsUpdate
+        ON orders(user_id, amount, mtsUpdate)`,
```

Comment:
where do we use this?

Reply:
It's for UI
![Screenshot from 2020-12-01 08-15-17](https://user-images.githubusercontent.com/16489235/100704049-75952700-33ad-11eb-9b8e-eebf4e4700d1.png)

Reply:
ok, lets remove this index